beep/featurize.py

```diff
@@ -137,7 +137,12 @@ def init_full_model(cls, processed_cycler_run, init_pred_cycle=10, mid_pred_cycl
         assert len(processed_cycler_run.summary) > final_pred_cycle, 'cycle count must exceed final_pred_cycle'
         cycles_to_average_over = 40  # For nominal capacity, use median discharge capacity of first n cycles
```

Comment:
Unrelated to the current PR, we were computing `nominal_capacity` for the fast charge project as median of discharge capacity over the first n=`cycles_to_average_over` cycles. This would lead to errors in the current projects (from looking at the profiles). Should we set nominal capacity based on a reference cycle, say 0 or 1?

Reply:
Yeah, I think that what we should do is calculate the nominal capacity based on the feature set. So for the fast charge feature set, we might take the median of the maximum capacity over the first 10 cycles or something.